app/routes/errors.py

```diff
@@ -37,13 +37,13 @@ def log_error(error, status_code):
     )
 
 
-def _render_error_page(status_code, template=None):
+def _render_error_page(status_code, template=None, **kwargs):
     handle_language()
     template = template or status_code
     using_edge = request.user_agent.browser == "edge"
 
     return (
-        render_template(template=f"errors/{template}", using_edge=using_edge),
+        render_template(template=f"errors/{template}", using_edge=using_edge, **kwargs),
```

Comment:
I realise this wasn't added in this pull request, but `using_edge` is no longer used and should be removed (https://github.com/ONSdigital/eq-questionnaire-runner/commit/d7079f94d03c5aa0c2ffb5823faa03fe1e8412ef#diff-b5a180a4c7c2be4b2d9e6472ed965d80).

Reply:
I meant to bring that up with you yesterday - I'll remove it now

Reply:
done in dd3818d